views/articles/show.ejs

```diff
@@ -0,0 +1,8 @@
+<% include ../partials/header.ejs %>
+
+
+<h1><%= articles.title %></h1>
+
+<p><%= articles.body %></p>
+
```

Comment:
add the footer partial link here, it will add the copy rite and close the body and html tags opened when you added the header partial